includes/admin/feedzy-rss-feeds-usage.php

```diff
@@ -213,6 +217,65 @@ public function get_usage_stats() {
 		return $stats;
 	}
 
+	/**
+	 * Record the import count per year week.
+	 * 
+	 * @return void
+	 * 
+	 * @since 5.0.8
+	 */
+	public function record_import_per_week() {
+		$data_time  = current_datetime();
+		$curr_week  = intval( $data_time->format( 'W' ) );
+		$curr_year  = intval( $data_time->format( 'Y' ) );
+		$curr_month = intval( $data_time->format( 'n' ) );
+
+		$imports_per_week = array();
+		$data             = $this->get_usage_data();
+		if ( is_array( $data['imports_per_week'] ) ) {
+			$imports_per_week = $data['imports_per_week'];
+		}
+
+		$should_add = true;
+		foreach ( $imports_per_week as &$import ) {
+			if (
+				$curr_week !== $import['week'] ||
+				$curr_year !== $import['year'] ||
+				$curr_month !== $import['month']
```

Comment:
Would be a bit more efficient if we check Year > Month > Week, in order of which is most likely to differ on the records.